### ISSUE TITLE
After changing cron expression - last_success date in database set as null

### ISSUE THREAD

**Issue by Dmitry-new** [kagkarlsson/db-scheduler#226]
We updated the cron expression for one job, which uses the last_success column from the Scheduled_Task table. And discovered that this field was set as null.

**Comment by kagkarlsson** [comment]
Yeah I agree, ideally the last_success/failure should be kept as it was
